api/src/db/migrations/v136_add_user_account_creation_source.ts

```diff
@@ -0,0 +1,747 @@
+import {
+  v135BusinessUser,
+  v135UserData,
+} from "@db/migrations/v135_use_interstate_logistics_and_transport_fields";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v135_to_v136 = (v135Data: v135UserData): v136UserData => {
+  return {
+    ...v135Data,
+    user: migrate_v135BusinessUser_to_v136BusinessUser(v135Data.user),
+    version: 136,
+  } as v136UserData;
+};
+
+export const migrate_v135BusinessUser_to_v136BusinessUser = (
+  v135BusinessUser: v135BusinessUser
+): v136BusinessUser => {
+  return {
+    ...v135BusinessUser,
+    accountCreationSource: "",
```

Comment:
Not sure how consistent we are in the code base about this but do we care about "" vs undefined?